### ISSUE TITLE
Error: Cannot read property 'type' of undefined

### ISSUE THREAD

**Issue by MatzeDS** [JetBrains/svg-mixer#59]
Hello,
I use "extract-svg-sprite-webpack-plugin" as described in the example and get the following error:
```
(node:10720) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'type' of undefined
    at Function.replaceInModuleSource (/home/user/Embedded/7490/HTML_vr9_build/node_modules/extract-svg-sprite-webpack-plugin/lib/utils/replacer.js:62:16)
    at s.cssModules.forEach.m (/home/user/Embedded/7490/HTML_vr9_build/node_modules/extract-svg-sprite-webpack-plugin/lib/plugin.js:66:32)
    at Array.forEach (<anonymous>)
    at sprite.symbols.forEach.s (/home/user/Embedded/7490/HTML_vr9_build/node_modules/extract-svg-sprite-webpack-plugin/lib/plugin.js:65:34)
    at Array.forEach (<anonymous>)
    at result.forEach (/home/user/Embedded/7490/HTML_vr9_build/node_modules/extract-svg-sprite-webpack-plugin/lib/plugin.js:63:32)
    at Array.forEach (<anonymous>)
    at compileSprites.then.result (/home/user/Embedded/7490/HTML_vr9_build/node_modules/extract-svg-sprite-webpack-plugin/lib/plugin.js:62:22)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
```

In cssModules of symbols are undefined values (line: 65):

`s.cssModules.forEach(m => {`

I use the following webpack plugins:
- VueLoaderPlugin
- MiniCssExtractPlugin
- HtmlWebpackPlugin

and loader for css:
- MiniCssExtractPlugin.loader
- css-loader
- SpritePlugin.cssLoader
- postcss-loader
- sass-loader

**Comment by alexbelij** [comment]
lib/utils/replacer.js
After string
`static replaceInModuleSource(module, replacements, compilation) {`
add
`if (typeof module !== 'object') return;`

**Comment by mauricevancooten** [comment]
It will be great if this plugin can work with CSS Modules

**Comment by Tommytto** [comment]
I have the same problem without css modules

**Comment by mauricevancooten** [comment]
Me too without CSS Modules

**Comment by Tommytto** [comment]
May be it will be helpful. I am using dynamic imports, mini-css-extract-plguin, sass, html webpack plugin

**Comment by Tommytto** [comment]
all tools have latest versions

**Comment by rasgor** [comment]
Same here

**Comment by GrushinEgor** [comment]
Same here

**Comment by kisenka** [comment]
Could you please create repo with minimal setup to demonstrate a problem (package.json, webpack config, SVG image and piece of your code). If you don't want to create a repository - you can create a [gist](http://gist.github.com) with multiple files.

**Comment by kisenka** [comment]
Fixed in `extract-svg-sprite-webpack-plugin@0.7.1`
